### ISSUE TITLE
[sub-issue] CRw: Language selection RM is not interpreted correctly.

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#301]
![image](https://cloud.githubusercontent.com/assets/7510842/5380411/1c3bbf44-8098-11e4-9744-f4585bc5eaf8.png)

It's not announced to a screen reader that "RM" is not selectable.
==> make it similar to "DE"  with a class="sr-only"

For better readability please complete the languages DE, FR, IT, RM, EN with the whole names
==> like above: class="sr-only"

**Comment by tonifisler** [comment]
Main issue is #340
